JetNews/app/src/main/java/com/example/jetnews/ui/interests/SelectTopicButton.kt

```diff
@@ -50,24 +51,30 @@ fun SelectTopicButton(
 
 @Composable
 private fun DrawSelectTopicButtonOn() {
```

Comment:
This composables previously were without layout, so now they have somewhere unnecessary layout above, consider removing it and pass modifiers here as a param

Reply:
Much cleaner, thx!